CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,9 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.58.2...master)
+
+## FxA Client
+- iOS: Added optional boolean argument `forceRefresh` to ignore caching for `fetchDevices`.
+- Added an optional `ttl` parameter to `getAccessToken` to limit the lifetime of the token. ([#3066](https://github.com/mozilla/application-services/pull/3066))
```

Comment:
There's some stuff here about `ttl` which doesn't seem quite right, possibly copy-pasta or a bag merge from a previous changelog?

Reply:
Copy pasta for sure. You saw nothing.